### DISCUSSION TITLE
katex not rendering all on title blogger

### DISCUSSION THREAD

**Discussion by ghost** [KaTeX/KaTeX#3587] [Q&A]
![image](https://user-images.githubusercontent.com/41530252/159229206-929fd5db-8bb9-41a2-84f7-debf8d17bb85.png)

hallo... i want to ask about rendering katex on title blogger.

i uses auto rendering katex with that code on my blogger template and place before </body>

<link crossorigin='anonymous' href='https://cdn.jsdelivr.net/npm/katex@0.15.3/dist/katex.min.css' integrity='sha384-KiWOvVjnN8qwAZbuQyWDIbfCLFhLXNETzBQjA/92pIowpC0d2O3nppDGQVgwd2nB' rel='stylesheet'/>
<script crossorigin='anonymous' defer='defer' integrity='sha384-0fdwu/T/EQMsQlrHCCHoH10pkPLlKA1jL5dFyUOvB3lfeT2540/2g6YgSi2BL14p' src='https://cdn.jsdelivr.net/npm/katex@0.15.3/dist/katex.min.js'/>
<script crossorigin='anonymous' defer='defer' integrity='sha384-+XBljXPPiv+OzfbB3cVmLHf4hdUFHlWNZN5spNQ7rmHTXpd7WvJum6fIACpNNfIR' src='https://cdn.jsdelivr.net/npm/katex@0.15.3/dist/contrib/auto-render.min.js'/>
<script>
    document.addEventListener(&quot;DOMContentLoaded&quot;, function() {
        renderMathInElement(document.body, {
          // customised options
          // &#8226; auto-render specific keys, e.g.:
          delimiters: [
              {left: &#39;$$&#39;, right: &#39;$$&#39;, display: true},
              {left: &#39;$&#39;, right: &#39;$&#39;, display: false},
              {left: &#39;\\(&#39;, right: &#39;\\)&#39;, display: false},
              {left: &#39;\\[&#39;, right: &#39;\\]&#39;, display: true}
          ],
          // &#8226; rendering keys, e.g.:
          throwOnError : false
        });
    });
</script>

but not all link title has rendering likes picture above. all equation except title is rendering. how to fix it ?

**Comment by edemaine** [comment]
Presumably the title is rendering *after* KaTeX runs (which according to your code, happens when the DOM content is loaded, but not after). How are you rendering your title? You could call auto-render afterwards, or when rendering the title, use `renderToString`.

**Reply by ghost** [reply]
> 

How are you rendering your title? when i see a code in blogger template...there is a function get link from feed then return a string as a title

You could call auto-render afterwards...i have same idea...i want call katex auto rendering in the end...but how to do it ?

when rendering the title, use renderToString...is renderToString can render string or text with delimeteres $ or $$? how can i do it, i cant get example in any website. i'm ever using katex.render, but still no rendering a text

thank for your help
